cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
this file is completely commented out? what's its use?

Reply:
These are part of the files auto-generated by Cypress.... no idea what it does, but it was there.